### ISSUE TITLE
neovide --wsl results in a frozen black window

### ISSUE THREAD

**Issue by lukeemhigh** [neovide/neovide#1803]
**Describe the bug**

Running `neovide --wsl` from PowerShell results in a frozen black windows that takes no input and can be closed only with Task Manager.

**To Reproduce**

1. Run `neovide --wsl`

**Expected behavior**

Invoking the `neovide --wsl` command results in a window running a working graphical frontend for my wsl-installed neovim

**Screenshots**

![screenshot](https://user-images.githubusercontent.com/113122350/225136488-af2f7233-8ab0-44c5-a872-7f12795a5955.png)

**Desktop:**

 - OS: Windows 11 Pro 22H2
 - Neovide Version 0.10.3
 - Neovim Version 0.9.0-dev

[Log file](https://github.com/neovide/neovide/files/10973753/neovide_rCURRENT.log)

**Additional context**

Neovide installed using scoop.

WSL Setup

```
Windows Subsystem for Linux Distributions:
Ubuntu-22.04 (Default)
docker-desktop-data
docker-desktop
```

Using default astronvim config files

**Comment by Ontonator** [comment]
I think I have the same issue, although I'm using `--remote-tcp` instead of `--wsl`. From what I can tell, it started at https://github.com/neovim/neovim/commit/fd2ece278b0941ec6673489e88868120e86b834a.

**Comment by Ontonator** [comment]
This seems to be fixed in https://github.com/neovide/neovide/commit/fadc48f0fc554c0e3a5c5f80ad288f9c57a70e6d.

**Comment by fredizzimo** [comment]
Fixed by https://github.com/neovide/neovide/pull/1802

**Comment by lukeemhigh** [comment]
> This seems to be fixed in [fadc48f](https://github.com/neovide/neovide/commit/fadc48f0fc554c0e3a5c5f80ad288f9c57a70e6d).

Yep, I can confirm, thx!
